lib/pages/list.js

```diff
@@ -64,6 +69,7 @@ module.exports = React.createClass({displayName: 'ListPage',
 
             <h2>Owned by you:</h2>
             {owned}
+        <PrSearcher controller={this.props.controller} history={this.props.history} />
```

Comment:
I would imagine having the searcher at the top of the page would be more useful.